src/Aspire.Hosting.Azure/Aspire.Hosting.Azure.csproj

```diff
@@ -40,6 +40,8 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Azure.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Azure.AppContainers" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Azure.AppService" />
```

Comment:
Avoid internals visible to for Aspire.Hosting.Azure.AppContainers and Aspire.Hosting.Azure.AppService and instead use that file as shared source across those 2 projects.